ibm/service/vpc/resource_ibm_is_vpc_address_prefix.go

```diff
@@ -21,6 +21,7 @@ const (
 	isVPCAddressPrefixHasSubnets = "has_subnets"
 	isVPCAddressPrefixDefault    = "is_default"
 	isAddressPrefix              = "address_prefix"
+	isVPCRouteDestinationCIDR    = "destination"
```

Comment:
isVPCRouteDestinationCIDR field is not used any where except in the validation and that validation identifier is "destination" and "destination" field is not in the schema. Can you please help me to understand this?

Reply:
This is an existing limitation in the code, current changes removes the deprecated resource/ds as part of same the declaration of the `destination` variable `isVPCRouteDestinationCIDR` is moved from the removed to used one.

We can resolve this incomplete usage of the variable, with current changes.